styles/styles.css

```diff
@@ -0,0 +1,495 @@
+@import url('https://fonts.googleapis.com/css?family=Archivo+Black|Merriweather:400,400i');
+
+@media screen and (max-width: 767px) {
+  .profiles {
+    width: 350px;
+  }
+}
+
+@media screen and (min-width: 768px) and (max-width: 991px) {
+  .profiles {
+    width: 700px;
+  }
+}
+
+@media screen and (min-width: 992px) {
+    .profiles {
+      max-width: 1600px;
+    }
+}
+
+@media screen and (min-width: 950px) {
+  .main-body {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    /*Note to instructors: I don't like this choice. It seems like there's a better way to do it but when I keep the .main-body on flex at all times, things overlap at smaller screen sizes. I haven't figured out a better soluation.*/
```

Comment:
Can you make an office hours? I'd love to help you out on this, and it's sometimes better to work through it together live.